app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(list_of_players: list[Player]) -> int:
```

Comment:
list_of_ prefix is redundant rename to players
list_of_elves -> elves
list_of_dwarves -> dwarves